opslevel/import_utils.go

```diff
@@ -53,6 +53,13 @@ func getTagsFromResource(client *opslevel.Client, resource opslevel.TaggableReso
 	return tags, diags
 }
 
+// hasTagFormat returns true if the given tag is formatted as '<key>:<value>'
+func hasTagFormat(tag string) bool {
```

Comment:
Why is `isTagValid` different from this?  is that function miss named?

Reply:
`isImportTagValid` would be a better name. I can add that in